testdata/message2/README.txt

```diff
@@ -0,0 +1,89 @@
+© 2024 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+
+TODO: the following only applies to C++
+
+The format of the JSON files in this directory follows the same format as `test-core.json`
+in the spec, described in:
+
+https://github.com/unicode-org/message-format-wg/blob/main/test/README.md
+
+The `parts` field is not used.
+
+# JSON extensions
+
+An additional `comment` field may be present, which is only for human readers.
+
+A "srcs" field, whose value is an array of strings, may be present instead
+of "src". The strings are concatenated to get the message.
+
+In the "params" field, a date parameter can be expressed as:
+{ "date": n }
+where n is a number representing a Unix timestamp.
+
+In the "params" field, a decimal string parameter can be expressed as:
+{ "decimal": s }
+where s is a string.
+
+Optional fields, "ignoreJava" and "ignoreCpp" can be used
+for tests currently expected to fail in the respective language.
+The field may have any value; if it's
+present, the test is ignored. (The value can be a comment explaining
+why it's expected to fail.)
+
+Tests in the `spec/` subdirectory are taken from https://github.com/unicode-org/message-format-wg/blob/main/test
+and need to be manually updated if the contents change upstream.
```

Comment:
put a reference to the ticket for making this automatic! 

and actually, it would be from CLDR not mfwg?

Reply:
Sort of -- if MFWG changes the spec tests, then CLDR would need to change to reflect those changes (for non-spec tests, CLDR is the source of truth). Clarified this in [1993b61](https://github.com/unicode-org/icu/pull/3050/commits/1993b61ab6d20c5f205933842ed94242e9ec96a8).